src/System.Runtime.Extensions/tests/AssemblyResolveTests/AssemblyResolveTests.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <ProjectGuid>ad83807c-8be5-4f27-85df-9793613233e1</ProjectGuid>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <CopyNuGetImplementations>false</CopyNuGetImplementations>
```

Comment:
We might even want it i n dir.props, since I believe that we don't need to copy this anywhere.